pkg/controllers/manager.go

```diff
@@ -32,16 +32,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/controller-runtime/pkg/webhook"
 )
 
 var (
 	scheme = runtime.NewScheme()
 )
 
 func init() {
-	log.PanicIfError(clientgoscheme.AddToScheme(scheme), "adding clientgo to scheme")
-	log.PanicIfError(apis.AddToScheme(scheme), "adding apis to scheme")
+	_ = clientgoscheme.AddToScheme(scheme)
```

Comment:
Do we not want to catch these errors?

Reply:
I don't believe this actually can error.